### ISSUE TITLE
Don't define specific font-size for p tag

### ISSUE THREAD

**Issue by Grsmto** [Neverbland/neverbuild#3]
We shouldn't specify a font-size for the `p` tag cause it adds specificity to our stylesheets (`_type.scss` line 67).

For example if you want to set the font-size of a div to `22px` you will have to do
`.my-div p {
    font-size: 22px;
}`

The browser default inherit from parent declared font-size, so this is not needed.

**Comment by vslio** [comment]
@Grsmto Agreed. I haven't cleaned up the mess. That line is most likely from an old project's base seeing it defaults at `14px`/`1.4rem` as well as other stuff like:

```
$base-font: sans-serif;
$alt-font: sans-serif;

.alt-font { font-family: $alt-font; }
```

**Comment by vslio** [comment]
Solved with 6ee10d510c9a99e86a106457f5d5f8f8a9d9ced9
